src/features/channels/components/channel-list/channel-list-item/ChannelListItem.tsx

```diff
@@ -108,7 +111,22 @@ const ChannelListItem = ({ channel, avatar = channel.channelName, selected = fal
           secondary={
             <Box display="flex">
               <Typography noWrap component="span" variant="caption" color="text.secondary">
-                {channel.messages.length ? channel.messages[0].decryptedMessage : ''}
+                {!!channel.messages.length && (
+                  <>
+                    {channel.messages[0].messageType === 'attachment' && (
+                      <div
+                        style={{
```

Comment:
Styles should not be in the component, fix it

Reply:
Go